packages/twenty-front/src/modules/object-record/record-table/components/RecordTableInternalEffect.tsx

```diff
@@ -33,7 +33,7 @@ export const RecordTableInternalEffect = ({
     callback: () => {
       leaveTableFocus();
     },
-    mode: ClickOutsideMode.comparePixels,
+    mode: ClickOutsideMode.compareHTMLRef,
```

Comment:
why this change? don't we have an issue with portals? (filter dropdown, field input) ?

Reply:
I changed it in a recent PR but this introduced the bug of double click to open